tests/accessor/accessor_exceptions.hpp

```diff
@@ -0,0 +1,379 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This file provides functions for tests on accessor exceptions.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_ACCESSOR_EXCEPTIONS_H
+#define SYCL_CTS_ACCESSOR_EXCEPTIONS_H
+
+#include "../../util/usm_helper.h"
+#include "accessor_common.h"
+#include "catch2/catch_test_macros.hpp"
+
+#include <type_traits>
+
+namespace accessor_exceptions_test {
+
+/**
+ * @brief Function helps to verify that constructors accessor really thrown
+ *        exceptions.
+ * @tparam AccType Current type of the accessor: generic_accessor,
+ *         local_accessor, or host_accessor
+ * @tparam DataT Current data type
+ * @tparam Dimension Dimension size
+ * @tparam Target Current target
+ * @tparam GetAccFunctorT Type of the functor that will be use to interact with
+ *         accessor
+ */
+template <accessor_tests_common::accessor_type AccType, typename DataT,
+          int Dimension, sycl::target Target, typename GetAccFunctorT>
+void check_exception(GetAccFunctorT construct_acc) {
```

Comment:
Do we really need this function for local_accessor if there are completely separate branches for each type of accessor and it called just once for it?

Reply:
Yes, I prefer to have the same ways to verify that exception has been thrown as for two another accessor classes